src/cpu/BaseCPU.py

```diff
@@ -315,3 +317,10 @@ def __init__(self, **kwargs):
                 self._uncached_interrupt_request_ports
                 + ["interrupts[0].int_requestor"]
             )
+
+    def addLoopPointProbe(self, targetpc, manager):
```

Comment:
Let's put this in the stdlib instead of in the SimObject description file. I'd like to start migrating things out of this file into the stdlib more generally :)